app/react/docker/app-templates/TemplateListSort/TemplateListSort.tsx

```diff
@@ -1,3 +1,7 @@
+import clsx from 'clsx';
```

Comment:
what's the difference of this file from `app/react/components/datatables/SortbySelector.tsx`

Reply:
It seems like the only difference is type of the filter options and values, not sure about the styles. Will leave this outside the scope of this ticket though as it could be worth making a storybook story for this too